.travis.yml

```diff
@@ -4,9 +4,9 @@ language: ruby
 matrix:
   include:
      - rvm: 2.0.0
-     - rvm: 2.1
-     - rvm: 2.2
-     - rvm: 2.3.0
+     - rvm: 2.1.10
```

Comment:
Hmm, I thought 2.1 and friends were supposed to pick the newest version, but if they don't your changes make sense.